xc7/fasm2bels/iob_models.py

```diff
@@ -116,45 +141,49 @@ def append_ibuf_iostandard_params(top, site, bel, possible_iostandards):
                 "IOSTANDARD setting provided for {} do not match"
                 "its counterpart decoded from the fasm".format(site.site.name)
             )
-            return
 
-        bel.parameters["IOSTANDARD"] = '"{}"'.format(iostandard)
+            print("Requested:")
```

Comment:
What is going on with the `print`'s?

Reply:
I wanted fasm2bels to be more informative when an IOSTANDARD mismatch occurs.

Reply:
Should, probably write to stderr?